jar-connector/src/main/java/com/sixsq/slipstream/factory/RunFactory.java

```diff
@@ -39,7 +33,8 @@
 
 public abstract class RunFactory {
 
-	public final Run createRun(Module module, User user) throws SlipStreamClientException {
+	public final Run
+	createRun(Module module, User user) throws SlipStreamClientException {
```

Comment:
Strange formatting. I would put `throws SlipStreamClientException {` in the second line instead of `createRun(Module module, User user)...`